integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4988,7 +4988,7 @@ msgid ""
 "Currently, mobile phone number is not displayed in the Integreat App. You "
```

Comment:
I was not sure, but I checked what DeepL, and it does seem that `Currently, the mobile phone number…` is more common than without the article in English as well. Could you add it to the source line and generate translations again?

Reply:
Pushed 😸